src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php

```diff
@@ -203,6 +203,47 @@ public function findByIds(array $ids): array
             ->getQuery()->getResult(Query::HYDRATE_ARRAY);
     }
 
+    /**
+     * @param int[]    $ancestryDbIds
+     * @param string   $entityType
+     * @param int|null $limit
+     * @param int      $offset
+     * @return array
+     */
+    public function findDescendantsByType(array $ancestryDbIds, string $entityType, ?int $limit, int $offset) : array
```

Comment:
yes, is used in ProgrammesAggregationService.php in multiple cases:
https://github.com/bbc/programmes-pages-service/blob/21b5352489199a35032b314d8f0822ed71057820/src/Service/ProgrammesAggregationService.php#L41